### PR TITLE
Clarification of acceptable use of "fail" and "disable"

### PR DESCRIPTION
**Pull request by rachel-mack**
# Pull Request Process

Please read the
[Style Guide Review Process](https://wiki.corp.mongodb.com/display/DE/Technical+Writer+Style+Guide+Review+Process)
wiki page.

Contributors should take the following actions:

- Request a PR review in #docs-style-guide Slack channel.
- Add the Style Guide team reps as reviewers.
- If any reviewer requests changes, address them and request another review.
- When you receive PR review approvals from all the Style Guide team reps,
  they will merge the PR and notify you of the merge in the Slack channel.

# Pull Request Description

JIRA URL: <https://jira.mongodb.org/browse/DOCSP-33817> (related to, but does not solve)

Staging URL:
 - https://docs-mongodbcom-staging.corp.mongodb.com/meta/docsworker-xlarge/DOCSP-33817-disable-fail/style-guide/writing/use-positive-statements/
 - https://docs-mongodbcom-staging.corp.mongodb.com/meta/docsworker-xlarge/DOCSP-33817-disable-fail/style-guide/terminology/alphabetical-terms/#f
 - https://docs-mongodbcom-staging.corp.mongodb.com/meta/docsworker-xlarge/DOCSP-33817-disable-fail/style-guide/terminology/alphabetical-terms/#d

Related Vale PR: <https://github.com/mongodb/mongodb-vale-action/pull/4>

### Use of "Fail"
Clarify use cases: "Fail" should not be used to describe the actions of an individual or company, but is acceptable to describe system performance. This should be stated more clearly in the style guide.

### Use of "Disable"

Clarify use cases:: "Disable" should be used only in cases that describe whether or not a system employs a feature. We should clarify alternatives more explicitly.

### CODE REVIEW

**Review by ccho-mongodb — APPROVED** [review]
LGTM, thanks for the update!

File: source/style-guide/includes/terms/d.rst [hunk 1]
```diff
@@ -109,8 +109,8 @@ D
           -
 
    disabled
-     :icon-fa4:`exclamation-triangle` Don't use *disabled* to describe
-     inactive commands, options, or buttons on the interface. Use
+     :icon-fa4:`exclamation-triangle` Use *inactive* to describe
+     commands, toggled options, or buttons on the interface. Use
```
Comment (corryroot) [inline source/style-guide/includes/terms/d.rst:113]:
```suggestion
     commands, options, toggles, or buttons in the interface. Use
```

File: source/style-guide/includes/terms/f.rst [hunk 2]
```diff
@@ -4,7 +4,10 @@ F
 .. glossary::
 
    fail to
-     :icon-fa4:`times-circle` Use *unable to* instead.
+     :icon-fa4:`times-circle` Use *unable to* instead when referring to an individual or
+     company.
```
Comment (corryroot) [inline source/style-guide/includes/terms/f.rst:8]:
```suggestion
     :icon-fa4:`times-circle` Use *unable to* when you refer to an individual or
     company. 
```

File: source/style-guide/includes/terms/f.rst [hunk 3]
```diff
@@ -4,7 +4,10 @@ F
 .. glossary::
 
    fail to
-     :icon-fa4:`times-circle` Use *unable to* instead.
+     :icon-fa4:`times-circle` Use *unable to* instead when referring to an individual or
+     company. 
+     
+     Use of *fail* is acceptable to describe the performance of a system.
```
Comment (corryroot) [inline source/style-guide/includes/terms/f.rst:10]:
```suggestion
     You may use *fail* to describe system performance.
```

File: source/style-guide/writing/use-positive-statements.txt [hunk 4]
```diff
@@ -31,8 +31,9 @@ and are typically shorter.
 Avoid Negative Words
 --------------------
 
-Try to avoid the following negative words, using instead the suggested
-alternatives. However, always be honest and transparent about issues.
+Try to avoid the following negative words, using instead the suggested alternatives,
+particularly in reference to the user or the company. However, always be honest and
+transparent about issues, and accurately describe a system's performance.
```
Comment (corryroot) [inline source/style-guide/writing/use-positive-statements.txt:36]:
```suggestion
particularly in reference to the user or the company. However, always communicate honestly and
transparently about issues, and accurately describe a system's performance.
```

**Review by corryroot — APPROVED** [review]
LGTM! I commented with some suggestions.

**Review by jeff-allen-mongo — APPROVED** [review]
LGTM!

**Review by lindseymoore — APPROVED** [review]
LGTM!

**Pull request merged**